doc/chapter-environment-variables.txt

```diff
@@ -9,3 +9,7 @@ CURL_CA_BUNDLE::
 TMPDIR::
         Tells Newsboat to use the specified directory for storing temporary files.
         If this variable doesn't exist, a default of /tmp will be used.
+
+BROWSER::
+        Tells Newsboat which browser to use when opening an article in the browser.
+        If this variable doesn't exist, a default of lynx(1) will be used.
```

Comment:
I'd put that block at the beginning of the file, so the entries are kept in alphabetic order—it makes it easier to find if you know what you're looking for.

The description turned out misleading; it makes one believe that setting the variable would override `browser` setting in the config, which it won't. Want to try again?

Reply:
I'd modify the first sentence like that: "Tells Newsboat what browser to use if there is no `browser` setting in the config file". The second sentence looks good to me.